src/main/java/com/mg105/use_cases/PlayerGetsTutorial.java

```diff
@@ -0,0 +1,50 @@
+package com.mg105.use_cases;
+
+import com.mg105.entities.GiveTutorial;
+import com.mg105.utils.TutorialTexts;
+
+import java.util.List;
+
+public class PlayerGetsTutorial {
+    private Boolean completed = false;
+    private List<String> tutorialPhases;  // Go through multiple phases of tutorial in order
+
+    private GiveTutorial tutorial;
+
+    public PlayerGetsTutorial(Boolean completed, List<String> tutorialPhases, GiveTutorial tutorial){
+        this.completed = completed;
+        this.tutorialPhases = tutorialPhases;
+        this.tutorial = tutorial;
+    }
+
+    /**
+     * Check if all required actions performed by player
+     */
+    public boolean isComplete(){
+        return tutorial.ActionPerformedGetter("moved") & tutorial.ActionPerformedGetter("attacked")
+            & tutorial.ActionPerformedGetter("usedItem");
+    }
```

Comment:
It seems redundant to have the completed attribute and having this method as well. Also these values are probably better off put as constants although that really isn't a big deal.